cypress/integration/plugins/ml-commons-dashboards/register_model_artifact_spec.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { MLC_URL } from '../../../utils/constants';
+
+if (Cypress.env('ML_COMMONS_DASHBOARDS_ENABLED')) {
+  describe('MLC - model artifact validation', () => {
+    const config = {
+      model_type: 'bert',
+      embedding_dimension: 768,
+      framework_type: 'sentence_transformers',
+      all_config:
+        '{"architectures":["BertModel"],"max_position_embeddings":512,"model_type":"bert","num_attention_heads":12,"num_hidden_layers":6}',
+    };
+
+    function fillForm() {
+      const uploadModelName = `traced_small_model-${new Date()
+        .getTime()
+        .toString(34)}`;
+
+      cy.get('input[name="name"]').type(uploadModelName);
+
+      // cy.fixture(
```

Comment:
Could we remove this annotations? We can consider add this fillForm as a cypress command.